migration_steps/shared/validation_mapping.json

```diff
@@ -466,6 +467,34 @@
           "identifiers": []
         }
     },
+    "client_status": {
```

Comment:
Hmm we should fix it so validation doesn't require a stub entry in validation_mapping.json in cases where the entity has a custom sql file

Reply:
Yeah not sure it's worth it for this project given where we're at but it's a valid point :)